src/graphql/graphql-api.php

```diff
@@ -0,0 +1,245 @@
+<?php
+/**
+ * GraphQL API
+ *
+ * @package vip-block-data-api
+ */
+
+namespace WPCOMVIP\BlockDataApi;
+
+defined( 'ABSPATH' ) || die();
+
+/**
+ * GraphQL API to offer an alternative to the REST API.
+ */
+class GraphQLApi {
+	/**
+	 * Initiatilize the graphQL API, if its allowed
```

Comment:
This comment could be better (either here or below), to explain that it hooks into the `graphql_register_types` action, which only fires if WPGraphQL is installed and enabled, and is further controlled by the `vip_block_data_api__is_graphql_enabled` filter.

Reply:
I have made this change as well